packages/stream-model-handler/src/__tests__/model-handler.test.ts

```diff
@@ -94,6 +94,52 @@ const FINAL_CONTENT = {
   },
 }
 
+const FINAL_CONTENT_WITH_ACCOUNT_DOCUMENT_VIEW: ModelDefinition = {
+  name: 'MyModel',
+  accountRelation: ModelAccountRelation.LIST,
+  schema: {
+    $schema: 'https://json-schema.org/draft/2020-12/schema',
+    type: 'object',
+    properties: {
+      owner: {
```

Comment:
I thought these view properties wouldn't show up in the schema itself, but it just metadata for the client library to expose the stream metadata with a given name?  @PaulLeCam

Reply:
Yes! That's right, sorry for the mistake.